src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SingleSpeedTalonDriveSubsystemWithTrajectorySubsystem.java

```diff
@@ -0,0 +1,216 @@
+package com.systemmeltdown.robotlib.subsystems.drive;
+
+import com.systemmeltdown.robotlib.CanIdMap;
+import com.systemmeltdown.robotlib.subsystems.drive.TalonGroup;
+import com.systemmeltdown.robotlib.util.Constants;
+
+import java.util.Map;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.sensors.PigeonIMU;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveOdometry;
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveWheelSpeeds;
+//import edu.wpi.first.wpilibj.drive.RobotDriveBase;
+
+public class SingleSpeedTalonDriveSubsystemWithTrajectorySubsystem extends SkidSteerDriveSubsystem {
+    /**
+	 * Whether or not the left talon group needs to be inverted Value: boolean
+	 */
+    public static final String CONFIG_IS_LEFT_INVERTED = "is_left_inverted";
+    
+    /**
+	 * Whether or not the right talon group needs to be inverted Value: boolean
+	 */
+    public static final String CONFIG_IS_RIGHT_INVERTED = "is_right_inverted";
+    
+    // The left-side drive encoder
+    private final Encoder m_leftEncoder =
+        new Encoder(CanIdMap.LEFT_ENCODER_PORTS[0], CanIdMap.LEFT_ENCODER_PORTS[1], 
+                    Constants.LEFT_ENCODER_REVERSED);
+    // The right-side drive encoder
+    private final Encoder m_rightEncoder =
+        new Encoder(CanIdMap.RIGHT_ENCODER_PORTS[0], CanIdMap.RIGHT_ENCODER_PORTS[1],
+                    Constants.RIGHT_ENCODER_REVERSED);
```

Comment:
You shouldn't be referencing Constants at all, that's external info to this library